.yamato/_projectcontext.yml

```diff
@@ -379,6 +379,8 @@ all_package_ci_project_2020.2:
       -  .yamato/_projectcontext.yml#test_all_project_windows_min_editor
       -  .yamato/_projectcontext.yml#test_all_project_macos_2020.2
       -  .yamato/_projectcontext.yml#test_all_project_macos_min_editor
+    triggers:
+        expression: push.branch eq "10.x.x/release" AND NOT pull_request.draft
```

Comment:
Maybe you want to include this backport too? https://github.com/Unity-Technologies/Graphics/pull/3254/files#diff-2c5ce6368332f1046c596f3ec903b63fd746f8961197be2b820a2d7203ff1c53R383

Reply:
ah yep, needs to go to 2021.1/staging first but yeah i'll include it in this